estela-web/package.json

```diff
@@ -3,39 +3,24 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@babel/core": "^7.0.0-0",
-    "@babel/plugin-syntax-flow": "^7.14.5",
-    "@babel/plugin-transform-react-jsx": "^7.14.9",
-    "@openapitools/openapi-generator-cli": "^2.3.9",
-    "@testing-library/dom": "^7.21.4",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^12.0.0",
-    "@types/react": "^17.0.14",
-    "@types/react-dom": "^17.0.14",
-    "@types/react-router-dom": "^5.1.8",
-    "antd": "^4.16.9",
-    "axios": "^0.21.2",
-    "eslint": "^8.24.0",
+    "antd": "^4.23.6",
     "follow-redirects": "^1.14.8",
     "hosted-git-info": "^2.8.9",
     "node-fetch": "^2.6.7",
     "nth-check": "^2.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "^5.0.1",
+    "react-router-dom": "^5.3.4",
     "sass": "^1.36.0",
-    "typescript": "^4.1.2",
+    "typescript": "^4.8.4",
     "web-vitals": "^1.0.1"
   },
   "scripts": {
-    "dev": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "build": "webpack --mode production",
+    "build:dev": "webpack --mode development",
+    "build:start": "cd dist && PORT=3333 npx serve",
+    "start": "webpack serve --open --mode development",
+    "start:live": "webpack serve --open --mode development --live-reload --hot",
```

Comment:
We have a `build` script in production mode, do we need a `start` script with the production flag as well?